docstore/awsdynamodb/query.go

```diff
@@ -110,7 +111,7 @@ func (c *collection) planQuery(q *driver.Query) (*queryRunner, error) {
 			cb = cb.WithFilter(filtersToConditionBuilder(q.Filters))
 			cbUsed = true
 		}
-		in := &dyn.ScanInput{TableName: &c.table}
+		in := &dyn.ScanInput{TableName: &c.table, ConsistentRead: aws.Bool(c.opts.ConsistentRead)}
```

Comment:
this would be easier to read if you had a newline before ConsistentRead

Reply:
Done.